index.html

```diff
@@ -1,91 +1,205 @@
         </div>
+      </div>
+
+      <div class="project-card">
+        <img src="./assets/Snapshoot Portfolio-3.png" alt="project card image" />
+        <div class="prject-card-info flex column">
+          <h2 class="project-title">Tonic</h2>
+          <div class="project-meta flex">
+            <p class="bold">Canopy</p>
+            <div class="round"></div>
+            <p>Backend</p>
+            <div class="round"></div>
+            <p>2015</p>
+          </div>
+          <div class="project-note">
+            <p>
+              A daily selection of privately personalized reads; no accounts
+              or sign-ups required.
+            </p>
+          </div>
+          <div class="project-tag">
+            <ul class="project-skills flex">
+              <li>html</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+          </div>
+          <button class="see-project-btn" type="button">See Project</button>
+        </div>
+      </div>
+
+      <div class="project-card">
+        <img src="./assets/Snapshoot Portfolio 3.svg" alt="project card image" />
+        <div class="prject-card-info flex column">
+          <h2 class="project-title">Tonic</h2>
+          <div class="project-meta flex">
+            <p class="bold">Canopy</p>
+            <div class="round"></div>
+            <p>Backend</p>
+            <div class="round"></div>
+            <p>2015</p>
+          </div>
+          <div class="project-note">
+            <p>
+              A daily selection of privately personalized reads; no accounts
+              or sign-ups required.
+            </p>
+          </div>
+          <div class="project-tag">
+            <ul class="project-skills flex">
+              <li>html</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+          </div>
+          <button class="see-project-btn" type="button">See Project</button>
+        </div>
+      </div>
+
+      <div class="project-card">
+        <img src="./assets/Snapshoot Portfolio-1.png" alt="project card image" />
+        <div class="prject-card-info flex column">
+          <h2 class="project-title">Tonic</h2>
+          <div class="project-meta flex">
+            <p class="bold">Canopy</p>
+            <div class="round"></div>
+            <p>Backend</p>
+            <div class="round"></div>
+            <p>2015</p>
+          </div>
+          <div class="project-note">
+            <p>
+              A daily selection of privately personalized reads; no accounts
+              or sign-ups required.
+            </p>
+          </div>
+          <div class="project-tag">
+            <ul class="project-skills flex">
+              <li>html</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+          </div>
+          <button class="see-project-btn" type="button">See Project</button>
+        </div>
+      </div>
     </section>
```

Comment:
- [ ] As requested by the previous reviewer and since its been listed in the project requirement, Please insure adding the **About me** section to your project.

![image](https://user-images.githubusercontent.com/68030297/219469088-5f2eb6e7-9815-45ae-95c9-a8223e4e34b7.png)